index.js

```diff
@@ -17,11 +18,24 @@ CoffeeScriptFilter.prototype.targetExtension = 'js'
 CoffeeScriptFilter.prototype.processString = function (string, srcFile) {
   var coffeeScriptOptions = {
     bare: this.bare,
-    literate: coffeeScript.helpers.isLiterate(srcFile)
+    literate: coffeeScript.helpers.isLiterate(srcFile),
+    sourceMap: !!this.sourceMap,
+
+    sourceFiles: [ srcFile ],
```

Comment:
I think this is reasonable until we can find a better solution. There is no reason for this PR to be blocked by this.

I'll make the change.